README.md

```diff
@@ -2,16 +2,29 @@
 
 The Quarkiverse GitHub organization provides repository hosting (including build, CI and release publishing setup) for Quarkus extension projects contributed by the community.
 
-You can browse the [Quarkiverse Wiki](https://github.com/quarkiverse/quarkiverse/wiki) to get more info and also find out how to get an [extension onboarded](https://github.com/quarkiverse/quarkiverse/wiki#getting-an-extension-onboarded).
+This repository serves several functions:
 
-# Run project locally
+- Host documentation for extension authors who wish to take advantage of the Quarkiverse Hub infrastructure. The docs are available on [this repository's wiki](https://github.com/quarkiverse/quarkiverse/wiki) and at [hub.quarkiverse.io](https://hub.quarkiverse.io).  
+- Host [issues] created by the [ecosystem CI](https://github.com/quarkusio/quarkus-ecosystem-ci#what-its-all-about)
 
-Install required dependencies via runing the following CMD
+## Onboarding an extension 
+
+Looking to onboard an extension? Here's the [quick link](https://github.com/quarkiverse/quarkiverse/wiki#getting-an-extension-onboarded). 
```

Comment:
We can change to hub.quarkiverse.io when the anchor links work there :)

Reply:
100% ... I started to change it, and then realised anchor links didn't work, and then started to raise an issue, and then realised that past-me had already raised https://github.com/quarkiverse/quarkiverse/issues/141. :)

I think it's easy to do, just bringing in the right plugin, but ...todo lists.